app/assets/stylesheets/components/_spinner-button.scss

```diff
@@ -17,7 +17,9 @@ lg-spinner-button {
     a,
     button:not([type]),
     [type='submit'],
-    [type='button'] {
+    [type='button'],
+    .usa-button:disabled.usa-button--active:not(.usa-button--unstyled, .usa-button--secondary, .usa-button--accent-cool, .usa-button--accent-warm, .usa-button--base, .usa-button--outline, .usa-button--inverse),
+    .usa-button--disabled.usa-button--active:not(.usa-button--unstyled, .usa-button--secondary, .usa-button--accent-cool, .usa-button--accent-warm, .usa-button--base, .usa-button--outline, .usa-button--inverse) {
```

Comment:
I'm not thrilled about this selector, but each of these `:not`'s from the original selector increase specificity, so we must match them in order to override the styles.

The plan is to avoid the `:not` selectors altogether by providing active, disabled styles for all button states, which may have helped avoid this issue in the first place, or at least will allow us to simplify the selector greatly.

That's tracked at LG-7269.